lib/Migrator/SyntacticMigratorPass.cpp

```diff
@@ -41,17 +206,30 @@ struct SyntacticMigratorPass::Implementation : public SourceEntityWalker {
 
   APIDiffItemStore DiffStore;
 
-  ArrayRef<APIDiffItem*> getRelatedDiffItems(ValueDecl *VD) {
-    llvm::SmallString<64> Buffer;
-    llvm::raw_svector_ostream OS(Buffer);
-    if (swift::ide::printDeclUSR(VD, OS))
-      return {};
-    // FIXME: overrides and conformances.
-    return DiffStore.getDiffItems(Buffer.str());
+  ArrayRef<APIDiffItem*> getDiffItems(ValueDecl *VD) {
```

Comment:
Can we inline this to `getRelatedDiffItems` so that it is clear `getRelatedDiffItems` is the only valid function users should call?